<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0074b4;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>698</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>698</width>
           <height>698</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;
border:5px solid #fff;
border-radius:10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="image">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>551</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(103, 183, 158);
border:2px solid #fff;
border-radius:10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="bt_upload">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Rockwell Extra Bold</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 5px solid #fff;
border-radius: 30px;
background-color: #d43f5d;
color:#fff;</string>
            </property>
            <property name="text">
             <string>Upload Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_process">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>241</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Rockwell Extra Bold</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 5px solid #fff;
border-radius: 30px;
background-color: #fc6e3d;
color:#fff;
</string>
            </property>
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Rockwell Extra Bold</family>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Number of objects detected</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="info_table">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QTableView {
                background-color: #f0f0f0;
                alternate-background-color: #e0e0e0;
                selection-background-color: #a0a0a0;
                selection-color: #ffffff;
                border: 1px solid #d0d0d0;
            }</string>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="bt_left">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #ffffff;
border:5px solid #fff;
border-radius:10px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/left.png</normaloff>icons/left.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_right">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #ffffff;
border:5px solid #fff;
border-radius:10px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/right.png</normaloff>icons/right.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSubir_Archivo">
   <property name="text">
    <string>Subir Archivo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
